atomicapp/nulecule/container.py

```diff
@@ -90,7 +90,7 @@ def pull(self, image, update=False):
 
         cockpit_logger.info('Skipping pulling docker image: %s' % image)
 
-    def extract(self, image, source, dest, update=False):
+    def extract(self, image, source, dest, update=False, file=False):
```

Comment:
Bit confused on the second function, since we don't detect explicit whether something needs to be updated, we just pass in True/False to the `update` variable.

So the second function is just going to call the first and override the previous information?